### DISCUSSION TITLE
Circulation Pump more than 3 entries for one day

### DISCUSSION THREAD

**Discussion by dawosch** [signalkraft/mypyllant-component#248] [Q&A]
I want to set more than 3 entries for the circulation pump for a day.
E.g. something like this:

Start: 7:00
End: 7:30

Start: 8:00
End: 8:30

Start: 9:00
End: 9:30

Start: 10:00
End: 10:30

and so on.
But it seems that this is also not possible like in the official myVaillant app. There you can also only set 3 intervals for one day.

**Comment by robdidu** [comment]
Hi, I'm new to HA and this implementation, but I think you can just do this timebased (so HA writes a new schedule to the heatpump every three hours) and the information given in this thread: https://github.com/signalkraft/mypyllant-component/discussions/212
From a diferent point of view: This will lead to a massive increase in power consumption, since you constantly heat up the tubes in your house. Just saying this since not everbody is aware of this. Good luck!
